stable/artifactory/README.md

```diff
@@ -144,7 +145,14 @@ The following tables lists the configurable parameters of the artifactory chart
 
 Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`.
 
-### Ingress TLS
+### Ingress and TLS
+To get Helm to create an ingress object with a hostname, add these two lines to your Helm command:
+```
+helm install --name artifactory \
+  --set ingress.enabled=true \
```

Comment:
Maybe worth adding --set ingress.tls.secretName="chart-example-tls" \

Reply:
Yes I will add that later on. I think the doc need a lot of small changes to be easy to use. This small change is by request from a company that found it hard to figure out how the syntax for the hosts argument worked, so I added that.